src/extension/ip_extraction.c

```diff
@@ -520,34 +524,44 @@ static bool _is_private_v6(const struct in6_addr *nonnull addr)
     static const struct {
         union {
             struct in6_addr base;
-            unsigned __int128 base_i;
+            uint64_t base_i[2];
         };
         union {
             struct in6_addr mask;
-            unsigned __int128 mask_i;
+            uint64_t mask_i[2];
         };
     } priv_ranges[] = {
         {
-            .base.s6_addr = {[15] = 1},
+            .base.s6_addr = {[15] = 1}, // loopback
             .mask.s6_addr = {[0 ... 15] = 0xFF}, // /128
         },
         {
-            .base.s6_addr = {0xFE, 0x80},
+            .base.s6_addr = {0xFE, 0x80}, // link-local
+            .mask.s6_addr = {0xFF, 0xC0}, // /10
+        },
+        {
+            .base.s6_addr = {0xFE, 0xC0}, // site-local
             .mask.s6_addr = {0xFF, 0xC0}, // /10
         },
+        {
+            .base.s6_addr = {0xFD}, // unique local address
+            .mask.s6_addr = {0xFF}, // /8
+        },
         {
             .base.s6_addr = {0xFC},
             .mask.s6_addr = {0xFE}, // /7
         },
     };
     // clang-format on
 
-    unsigned __int128 addr_i;
-    memcpy(&addr_i, addr->s6_addr, sizeof(addr_i));
+    uint64_t addr_i[2];
```

Comment:
We can use `__int128` here, it's just that the tracer doesn't allow us to use gnu extensions so we had to remove it there.

Reply:
Done